src/auth.js

```diff
@@ -0,0 +1,13 @@
+export function setLoginInfo(data) {
+  localStorage.setItem('renew_id', data.renew_id)
+  localStorage.setItem('access_token', data.access_token)
```

Comment:
Shouldn't you update api config here?

Reply:
What do you mean by update api config? :thinking:

Reply:
The Authorization header.